script/UpdateCBridgeFacetPackedToken.s.sol

```diff
@@ -0,0 +1,31 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity ^0.8.17;
```

Comment:
Also I see a problem with the naming of the deploy & update script.
The scripts should always be named Deploy<FacetName> and Update<FacetName>, otherwise the deploy scripts will probably run into errors.

If you have different versions of these scripts, I would still combine them into one and then pass a bool (or whatever) parameter that defines which mode/features to be used.

Reply:
The scripts are planned to be executed manually if needed, so I used different names. My idea would be that the master deploy script ignores them if the name does not match the facet name exactly.

Reply:
I do not fully understand why we are creating such "exceptional" cases if we could with minimum effort align them with our established procedures, such as suggested above.

I understand your opinion but it doesnt improve the maintainability of our code base and I dont see any strong reason for such an exception.

And lastly, I feel that we are not dealing correctly with review comments here. They are not resolved but the PR is merged anyway.
Same as with the comment above. I think I had a fair point there (about the license identifier) but the comment does not get resolved (e.g. by creating a new JIRA task for it), instead it's ignored or deemed out of scope and will just get forgotten that way. I feel we could collaborate better here.